TeachingRecordSystem/src/TeachingRecordSystem.Core/ReferenceDataCache.cs

```diff
@@ -160,16 +160,16 @@ public async Task<AlertCategory> GetAlertCategoryById(Guid alertCategoryId)
         return alertCategories.Single(ac => ac.AlertCategoryId == alertCategoryId, $"Could not find alert category with ID: '{alertCategoryId}'.");
     }
 
-    public async Task<AlertType> GetAlertTypeById(Guid alertTypeId)
+    public async Task<AlertType[]> GetAlertTypes(bool includeActiveOnly = false)
```

Comment:
We've got prior art for calling this `activeOnly`
```suggestion
    public async Task<AlertType[]> GetAlertTypes(bool activeOnly = false)
```